app/experimenter/nimbus-ui/src/components/PageEditAudience/index.tsx

```diff
@@ -76,7 +76,19 @@ const PageEditAudience: React.FunctionComponent<RouteComponentProps> = () => {
   }, []);
 
   return (
-    <AppLayoutWithExperiment title="Audience" testId="PageEditAudience">
+    <AppLayoutWithExperiment
+      title="Audience"
+      testId="PageEditAudience"
+      redirect={({ status }) => {
+        if (status.review) {
+          return "request-review";
```

Comment:
If the experiment is in review, the `/edit` path redirects to `edit/overview` and none of the `edit/[whatever]` URLs redirect to request review. Try out `https://localhost/nimbus/inverse-cohesive-structure/edit`

Reply:
This issue (as discussed in Slack I think was the accepted state and not the review state) should be addressed now.